include/lsst/afw/geom/XYTransform.h

```diff
@@ -52,7 +52,11 @@ class XYTransform : public daf::base::Citizen {
     typedef afw::geom::AffineTransform AffineTransform;
 
     explicit XYTransform();
-    virtual ~XYTransform() {}
+    XYTransform(XYTransform const &);
+    XYTransform(XYTransform &&);
+    XYTransform &operator=(XYTransform const &);
+    XYTransform &operator=(XYTransform &&);
```

Comment:
I think it would be best to leave it explicit, so that the reader can see that `XYTransform` has an assignment operator. Otherwise, it's easy to just assume the behavior is what you want it to be; I ran into the opposite problem (assuming a class was assignable when it wasn't) several times while making these changes.

Reply:
That's reasonable too.  I'd open a ticket to `delete` it later, but I think we're hoping to get rid of `XYTransform` once the conversion to `Transform` is complete, so it's probably not worth the bother.